resources/models/resource.py

```diff
@@ -347,6 +377,12 @@ def can_make_reservations(self, user):
     def can_approve_reservations(self, user):
         return self.is_admin(user) and user.has_perm('can_approve_reservation', self.unit)
 
+    def access_code_enabled(self):
```

Comment:
I would name this `is_access_code_enabled()` to be consistent with other methods (start with a verb).